lib/screen/account/select_account_page.dart

```diff
@@ -78,22 +78,12 @@ class _SelectAccountScreenState extends State<SelectAccountScreen> {
             ),
             Padding(
               padding: ResponsiveLayout.pageHorizontalEdgeInsets,
-              child: PostcardButton(
+              child: PostcardAsyncButton(
                 text: "continue".tr(),
                 fontSize: 18,
-                isProcessing: _isConfirming,
-                onTap: _selectedAddress != null
-                    ? () async {
-                        setState(() {
-                          _isConfirming = true;
-                        });
-                        await widget.onConfirm(_selectedAddress!);
-                        setState(() {
-                          _isConfirming = false;
-                        });
-                      }
-                    : null,
-                color: const Color.fromRGBO(79, 174, 79, 1),
+                enabled: _selectedAddress != null,
+                onTap: () => widget.onConfirm(_selectedAddress!),
```

Comment:
should we add await for `widget.onConfirrm`?

Reply:
no need for that

Reply:
the function already declare future